querydsl-docs/src/main/docbook/en-US/content/general/codegen.xml

```diff
@@ -242,6 +242,116 @@ import com.querydsl.core.annotations.Config;
 </project>
 ]]></programlisting>
 
+    <para>Alternatively <code>maven-compiler-plugin</code> can be used as well:
```

Comment:
could you also document what this will do? It looks like it will hook APT directly into the compilation. Also I think this configuration doesn't add the generated sources folder to the project source roots, or does it?